### ISSUE TITLE
crash while using 'swupd search'

### ISSUE THREAD

**Issue by kad** [clearlinux/swupd-client#923]
**Describe the bug**
SIGSEGV crash of client.

**To Reproduce**
Steps to reproduce the behavior
```
$ docker run --rm -it clearlinux
root@11479089d214 / # swupd update
...(update to current latest state the image)...
root@11479089d214 / # swupd update
Update started.
Version on server (29440) is not newer than system version (29440)
Update complete. System already up-to-date at version 29440
root@11479089d214 / # swupd search -i --time
Deprecated option -t was renamed. Prefer using -T or --top.

Segmentation fault (core dumped)
```

**Expected behavior**
No SIGSEGV crashes.

**Environment (please complete the following information):**
 - Clear Linux OS Version: Installed version: 29440
 - Platform: docker

**Comment by castulo** [comment]
I was able to reproduce this problem with swupd release 3.19.0.
This bug has already been fix in the next release.

**Comment by castulo** [comment]
This was fixed in swupd version 3.20.0. I'm closing this issue.
Feel free to reopen if you still see this issue. 

Note: the `swupd search` command was renamed to `swupd search-file`.
